README.md

```diff
@@ -82,55 +82,26 @@ _(This is where you begin the server setup instructions)_
 6. Create a virtual host & WSGI file for the Apache server to display the Django application by creating a new .conf file.
 
 	`sudo vim /etc/apache2/sites-available/001-devcycle.conf`.
-7. Copy and paste the following, you may edit these values if desired (such as where to collect static files).
+7. Create the apache conf file for mod_wsgi hosting.
 
 ```
-WSGIPythonPath usr/local/devcycle
-
-<VirtualHost *:80>
-        ServerName devcycle.se.rit.edu
-        ServerAlias devcycle.se.rit.edu
-        ServerAdmin someaddress@example.com
-
-        Alias /static /var/www/static/
-
-        <Directory /public/static/>
-                Order allow,deny
-                Allow from all
-        </Directory>
-
-        WSGIDaemonProcess devcycle processes=2 threads=15 display-name=%{GROUP} python-path=/usr/local/devcycle/
-        WSGIProcessGroup devcycle
-        WSGIScriptAlias / /usr/local/devcycle/dataCollection/wsgi.py
-
-        <Directory /usr/local/devcycle/dataCollection>
-                <Files wsgi.py>
-                Order deny,allow
-                Allow from all
-                </Files>
-        </Directory>
-</VirtualHost>
+sudo cp /usr/local/devcycle/001-devcycle.conf.template /etc/apache2/sites-available/001-devcycle.conf
+sudo vim /etc/apache2/sites-available/001-devcycle.conf
 ```
 
-* Create and Change the application settings:
+* Modify the Server alias to match your domain: `ServerAlias YOURSUBDOMAIN.se.rit.edu`
+
+
+8. Create and Change the application settings:
```

Comment:
Also, the "Configure the Analysis Dashboard" section calls out more changes to this file.  May be outside the scope of this PR, but it would be nice to see those combined.